src/sparsevector.jl

```diff
@@ -85,15 +85,17 @@ _unsafe_unfix(s::FixedSparseVector) = SparseVector(length(s), parent(nonzeroinds
 
 # Define an alias for a view of a whole column of a SparseMatrixCSC. Many methods can be written for the
 # union of such a view and a SparseVector so we define an alias for such a union as well
-const SparseColumnView{Tv,Ti}  = SubArray{Tv,1,<:AbstractSparseMatrixCSC{Tv,Ti},Tuple{Base.Slice{Base.OneTo{Int}},Int},false}
-const SparseVectorView{Tv,Ti}  = SubArray{Tv,1,<:AbstractSparseVector{Tv,Ti},Tuple{Base.Slice{Base.OneTo{Int}}},false}
-const SparseVectorUnion{Tv,Ti} = Union{AbstractCompressedVector{Tv,Ti}, SparseColumnView{Tv,Ti}, SparseVectorView{Tv,Ti}}
-const AdjOrTransSparseVectorUnion{Tv,Ti} = LinearAlgebra.AdjOrTrans{Tv, <:SparseVectorUnion{Tv,Ti}}
-const SVorFSV{Tv,Ti} = Union{SparseVector{Tv,Ti},FixedSparseVector{Tv,Ti}}
+const SparseColumnView = SubArray{<:Any,1,<:AbstractSparseMatrixCSC,Tuple{Base.Slice{Base.OneTo{Int}},Int},false}
+const SparseVectorView = SubArray{<:Any,1,<:AbstractSparseVector,Tuple{Base.Slice{Base.OneTo{Int}}},false}
+const SparseVectorUnion = Union{AbstractCompressedVector, SparseColumnView, SparseVectorView}
```

Comment:
Why were these changes needed? I know this is/should be internals but a lot of packages seem to use these. If we need new definitions, maybe use new names and keep the old ones?

Reply:
If there's a lot of packages that use it, I can revert these changes. The fact that the types were parameterized caused lots of pain in multiple dispatch internally. I should've run pkgeval on the bump PR. 😢

Reply:
But you can define
```
SparseVectorUnion2 = ...
```
and use that internally, just so the old names still exist for the "outside world" I mean.

Reply:
Yes, I'm already done with that locally. ;-)